src/stylesheets/components/_list-styles.scss

```diff
@@ -148,6 +152,15 @@ dl, dt, dd {
   }
 }
 
+/*--- Status banner ---*/
+
+.status--banner__beta li {
```

Comment:
Check the difference in the browser between this and the cookie status--banner, it is quite significant.